src/EnergyPlus/DualDuct.cc

```diff
@@ -235,23 +217,18 @@ namespace DualDuct {
         int ADUNum;                           // loop control to search Air Distribution Units
         Real64 DummyOAFlow(0.0);
 
-        state.dataDualDuct->NumDualDuctConstVolDampers =
-            state.dataInputProcessing->inputProcessor->getNumObjectsFound(state, state.dataDualDuct->cCMO_DDConstantVolume);
-        state.dataDualDuct->NumDualDuctVarVolDampers =
-            state.dataInputProcessing->inputProcessor->getNumObjectsFound(state, state.dataDualDuct->cCMO_DDVariableVolume);
-        state.dataDualDuct->NumDualDuctVarVolOA =
-            state.dataInputProcessing->inputProcessor->getNumObjectsFound(state, state.dataDualDuct->cCMO_DDVarVolOA);
-        state.dataDualDuct->NumDDAirTerminal =
-            state.dataDualDuct->NumDualDuctConstVolDampers + state.dataDualDuct->NumDualDuctVarVolDampers + state.dataDualDuct->NumDualDuctVarVolOA;
+        int NumDualDuctConstVolDampers = state.dataInputProcessing->inputProcessor->getNumObjectsFound(state, cCMO_DDConstantVolume);
+        int NumDualDuctVarVolDampers = state.dataInputProcessing->inputProcessor->getNumObjectsFound(state, cCMO_DDVariableVolume);
+        state.dataDualDuct->NumDualDuctVarVolOA = state.dataInputProcessing->inputProcessor->getNumObjectsFound(state, cCMO_DDVarVolOA);
+        state.dataDualDuct->NumDDAirTerminal = NumDualDuctConstVolDampers + NumDualDuctVarVolDampers + state.dataDualDuct->NumDualDuctVarVolOA;
         state.dataDualDuct->dd_airterminal.allocate(state.dataDualDuct->NumDDAirTerminal);
         state.dataDualDuct->UniqueDualDuctAirTerminalNames.reserve(state.dataDualDuct->NumDDAirTerminal);
```

Comment:
Not a big ass fan of these unordered_maps for checking name uniqueness, just use the original list, no? How many of these objects do you expect in a typical model?